src/vs/gitpod/browser/workbench/workbench.html

```diff
@@ -101,9 +152,9 @@
 	</body>
 
 	<!-- Startup (do not modify order of script tags!) -->
-	<script type="text/javascript" src="/_supervisor/frontend/main.js" onerror="onVsCodeWorkbenchError(event)" charset="utf-8"></script>
-	<script src="./static/out/vs/loader.js" onerror="onVsCodeWorkbenchError(event)" ></script>
-	<script src="./static/out/vs/webPackagePaths.js" onerror="onVsCodeWorkbenchError(event)" ></script>
+	<script type="text/javascript" src="/_supervisor/frontend/main.js" onerror="onVsCodeWorkbenchError(event)" charset="utf-8" crossorigin="anonymous"></script>
```

Comment:
@mustard-mh `crossorigin="anonymous"` it does not seem you are making use of it, i.e. idea was to add it if we want to collect information about script errors

Reply:
If we add this, `supervisor frontend` can use it too